<launch>

	<param name="cluedo_path_where" type="string" value="$(find robocluedo)/config/cluedo_where.txt" />
	<param name="cluedo_path_what" type="string" value="$(find robocluedo)/config/cluedo_what.txt" />
	<param name="cluedo_path_who" type="string" value="$(find robocluedo)/config/cluedo_who.txt" />
	
	<!-- <node pkg="turtlesim" name="turtlesim" type="turtlesim_node" /> -->
	<node pkg="robocluedo" name="cluedo_oracle" type="cluedo_oracle" output="screen"></node>
	
</launch>
